xchange-blockchain/src/main/java/org/knowm/xchange/blockchain/service/BlockchainAccountService.java

```diff
@@ -0,0 +1,164 @@
+package org.knowm.xchange.blockchain.service;
+
+import org.knowm.xchange.blockchain.BlockchainAdapters;
+import org.knowm.xchange.blockchain.BlockchainAuthenticated;
+import org.knowm.xchange.blockchain.BlockchainErrorAdapter;
+import org.knowm.xchange.blockchain.BlockchainExchange;
+import org.knowm.xchange.blockchain.dto.BlockchainException;
+import org.knowm.xchange.blockchain.dto.account.BlockchainAccountInformation;
+import org.knowm.xchange.blockchain.dto.account.BlockchainFees;
+import org.knowm.xchange.blockchain.params.BlockchainFundingHistoryParams;
+import org.knowm.xchange.blockchain.params.BlockchainWithdrawalParams;
+import org.knowm.xchange.client.ResilienceRegistries;
+import org.knowm.xchange.currency.Currency;
+import org.knowm.xchange.currency.CurrencyPair;
+import org.knowm.xchange.dto.account.*;
+import org.knowm.xchange.exceptions.NotYetImplementedForExchangeException;
+import org.knowm.xchange.instrument.Instrument;
+import org.knowm.xchange.service.account.AccountService;
+import org.knowm.xchange.service.trade.params.*;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static org.knowm.xchange.blockchain.BlockchainConstants.*;
+
+public class BlockchainAccountService extends BlockchainAccountServiceRaw implements AccountService {
+
+    public BlockchainAccountService(BlockchainExchange exchange, BlockchainAuthenticated blockchainApi, ResilienceRegistries resilienceRegistries) {
+        super(exchange, blockchainApi, resilienceRegistries);
+    }
+
+    @Override
+    public AccountInfo getAccountInfo() throws IOException {
+        try {
+
+            Map<String, List<BlockchainAccountInformation>> account = this.getAccountInformation();
+            List<Wallet> wallets = new ArrayList<>();
+
+            for (Map.Entry<String, List<BlockchainAccountInformation>> entry : account.entrySet()) {
+                List<Balance> balances = entry.getValue().stream().map(bal -> new Balance(bal.getCurrency(),
+                                bal.getTotal(),
+                                bal.getAvailable()))
+                        .collect(Collectors.toList());
+                wallets.add(Wallet.Builder.from(balances).id(entry.getKey()).build());
+            }
+            return new AccountInfo(wallets);
+        } catch (BlockchainException e) {
+            throw BlockchainErrorAdapter.adapt(e);
+        }
+    }
+
+    @Override
+    public String withdrawFunds(Currency currency, BigDecimal amount, String address) throws IOException {
```

Comment:
I would put a link or a @see to the method they can use

Reply:
It is indicated as follows
```
/**
     * For more information see {@link String withdrawFunds(WithdrawFundsParams params)}
     */
    @Override
    public String withdrawFunds(Currency currency, BigDecimal amount, String address) throws IOException {
        throw new NotYetImplementedForExchangeException(NOT_IMPLEMENTED_YET);
    }

    /**
     * For more information see {@link String withdrawFunds(WithdrawFundsParams params)}
     */
    @Override
    public String withdrawFunds(Currency currency, BigDecimal amount, AddressWithTag address) throws IOException {
        throw new NotYetImplementedForExchangeException(NOT_IMPLEMENTED_YET);
    }
```